src/main/kotlin/com/dansplugins/factionsystem/map/builders/FactionInfoBuilder.kt

```diff
@@ -0,0 +1,92 @@
+package com.dansplugins.factionsystem.map.builders
+
+import com.dansplugins.factionsystem.MedievalFactions
+import com.dansplugins.factionsystem.faction.MfFaction
+import java.text.DecimalFormat
+import java.text.DecimalFormatSymbols
+import kotlin.math.floor
+import kotlin.math.roundToInt
+
+/**
+ * A builder class for creating faction information in HTML format.
+ *
+ * @param plugin The MedievalFactions plugin instance.
+ */
+class FactionInfoBuilder(private val plugin: MedievalFactions) {
+
+    private val decimalFormat = DecimalFormat("0", DecimalFormatSymbols.getInstance(plugin.language.locale))
+
+    /**
+     * Builds the faction information as an HTML string.
+     *
+     * @param faction The faction for which the information is being built.
+     * @return The faction information in HTML format.
+     */
+    fun build(faction: MfFaction): String {
+        val factionService = plugin.services.factionService
+        val relationshipService = plugin.services.factionRelationshipService
+        val claimService = plugin.services.claimService
+        val playerService = plugin.services.playerService
+        return buildString {
+            append("<h1>${faction.name}</h1>")
+            if (plugin.config.getBoolean("showDescription")) {
+                append("<h2>Description</h2>")
+                append(faction.description)
+            }
+            if (plugin.config.getBoolean("showMembers")) {
+                append("<h2>Members (${faction.members.size})</h2>")
+                append(
+                    faction.members.groupBy { it.role }.map { (role, members) ->
+                        """
+                <h3>${role.name} (${faction.members.count { it.role.id == role.id }})</h3>
+                ${members.joinToString { member -> playerService.getPlayer(member.playerId)?.name ?: plugin.language["UnknownPlayer"] }}
+                        """.trimIndent()
+                    }.joinToString("<br />")
+                )
+            }
+            if (plugin.config.getBoolean("showLiege")) {
```

Comment:
Same for the rest of the booleans retrieved in this class; they are missing the `dynmap.` prefix

Reply:
Addressed